src/flows/components/FlowPipe.tsx

```diff
@@ -12,15 +13,17 @@ const FlowPipe: FC<FlowPipeProps> = ({id}) => {
   return useMemo(
     () => (
       <PipeProvider id={id}>
-        <Pipe
-          Context={props => {
-            const _props = {
-              ...props,
-              id,
-            }
-            return createElement(FlowPanel, _props)
-          }}
-        />
+        <EditorProvider id={id}>
```

Comment:
only panels that have a monaco editor have the concept of having an editor, and only the flux editor has the ability to inject code into the monaco editor. this context would be better scoped to the rawfluxquery panel type and not to all panels

Reply:
done.